source/intro.md

```diff
@@ -586,7 +584,7 @@ with multiple kinds of `category`. The data frame
 `aboriginal_lang` contains only 67 rows, and looks like it only contains Aboriginal languages.
 So it looks like the `loc[]` operation gave us the result we wanted!
 
-### Using `sort_values` and `head` to select rows by ordered values
+## Using `sort_values` and `head` to select rows by ordered values
```

Comment:
This didn't fit in the "using loc and []" subsection, so I moved it out to its own section